main.py

```diff
@@ -15,19 +14,18 @@
 PROJECT_FILE = ''
 
 
-def do_calculation(constraint_property_id: str, xmlreader: XMLReader) -> str:
-    constraint_property = xmlreader.find_constraint_property(constraint_property_id)
-
+def do_calculation(constraint_property_id: str, xmlreader: XMLReader) -> [str, float]:
```

Comment:
How can this be a str

Reply:
Fixed with https://github.com/Squiddrone/VPParametricsSolver/pull/2/commits/4c16266a6748f824b112cf3f8e9ee007f8f268ff